mods/ra/maps/allies-05c/allies05c.lua

```diff
@@ -184,7 +184,7 @@ FreeTanya = function()
 
 	if TanyaType == "e7.noautotarget" then
 		Trigger.AfterDelay(DateTime.Seconds(1), function()
-			Media.DisplayMessage(UserInterface.Translate("tanya-rules-of-engagement"), "Tanya")
+			Media.DisplayMessage(UserInterface.Translate("tanya-rules-of-engagement"), UserInterface.Translate("tanya"))
```

Comment:
what do you thing, would it be better to reuse actor names? `actor-e7.name`

Reply:
should at least add comments of the new uses of the translations keys in en.ftl

Reply:
> what do you thing, would it be better to reuse actor names? `actor-e7.name`

I'm practically just a mapper but I wouldn't mind. I referenced the same key through tanya.TooltipName in Negotiations, and the guide also used their tooltip name.

Reply:
Her name was localized for the German version https://cnc.fandom.com/de/wiki/Tanja_Adams

Reply:
People who might translate this would have trouble with a cryptic key like `actor-e7.name` so this is a bad idea.